worker/src/RTC/Transport.cpp

```diff
@@ -1914,9 +1916,13 @@ namespace RTC
 						continue;
 					}
 
+					rtt = consumer->GetRtt();
```

Comment:
What about the commented block above? Shouldn't we also pass RR when it belongs to the probator?

Reply:
correct.

Reply:
In fact the API in `RtpTransportControllerSend` expects a list of reports, not just the one of the probator (as suggested in the commented block above) and the libwebrtc code does not distinguish between probator or other streams in order to provide with the RR's. I'll consider the probator consumer `rtt` as well.

Reply:
Indeed there is no consumer for probation.

Reply:
comment block removed.

Reply:
There is no consumer for probator, right, but we should be able to read RR from probator and pass it to the bwe, am I wrong?

Reply:
oh, that's right. I've just ignore probator RR...

Reply:
> There is no consumer for probator, right, but we should be able to read RR from probator and pass it to the bwe, am I wrong?

It's done already.

Reply:
Where?

Reply:
after the loop: https://github.com/versatica/mediasoup/pull/464/files#diff-d9a2ff41c9bac4d9444a0ea711677398R1906

Reply:
If the RR belongs to the probator we do `continue` so `rtt` is 0., It is ok?

Reply:
the whole RTCP packet is provided after iterating.

Reply:
We take the RTT from one of the Consumers, probator does no have any so it's not considered for the RTT.

Reply:
> If the RR belongs to the probator we do continue so rtt is 0., It is ok?

It's a continue, not a break, we keep iterating the Receiver Reports.